common/djangoapps/third_party_auth/samlproviderconfig/tests/test_samlproviderconfig.py

```diff
@@ -43,3 +43,21 @@ def test_create_one_config(self):
         providerconfig = SAMLProviderConfig.objects.get()
         self.assertEqual(providerconfig.slug, 'test-slug')
         self.assertEqual(providerconfig.name, 'name-of-config')
+
+    def test_update_one_config(self):
+        # PUT auth/saml/v0/providerconfig/{id}/ -d data
+        url = reverse('samlproviderconfig-list')
+        data = {
+            'entity_id': 'id',
+            'metadata_source': 'http://test.url',
+            'name': 'name-of-config',
+            'enabled': 'true',
+            'slug': 'test-slug'
+        }
+        self.assertEqual(SAMLProviderConfig.objects.count(), 0)
+        response = self.client.post(url, data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(SAMLProviderConfig.objects.count(), 1)
+        providerconfig = SAMLProviderConfig.objects.get()
+        self.assertEqual(providerconfig.slug, 'test-slug')
+        self.assertEqual(providerconfig.name, 'name-of-config')
```

Comment:
this is really dumb and minor, but not sure if you are aware or not - there is a setting you can enable in VSCode that will automatically add a new line to files on save so you don't get the little red circle there on your last line.
The setting is "Render Final Newline".

Reply:
Thanks! It seems this setting is on for me and yet I ended up with this situation, wonder how ...

Reply:
Perhaps it's 'Insert Final Newline' ? I see that one is off for me

Reply:
yes this last one worked! Thanks...

Reply:
ah yes sorry I didn't realize there were two of those and forgot I had both enabled. hah. Good find!